src/main/java/mate/academy/dao/BookDaoimpl.java

```diff
@@ -0,0 +1,121 @@
+package mate.academy.dao;
+
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Book;
+
+@Dao
+public class BookDaoimpl implements BookDao {
+    @Override
+    public Book create(Book book) {
+        String sql = "INSERT INTO book(title, price) VALUES (?, ?)";
```

Comment:
The table name should be `books` instead of `book` to follow common naming conventions and match the checklist item.